models.py

```diff
@@ -778,6 +778,10 @@ def url(self):
     def count(cls, session):
         return session.query(cls).count()
 
+    @classmethod
+    def from_user_id(cls, session, user_id):
```

Comment:
We have an established pattern of `from_user`
```suggestion
    def from_user(cls, session, user):
        return session.query(cls).filter(cls.user_id == user.id).first()
```

Reply:
changed to this:

def from_user(cls, session, user):
        return session.query(cls).filter(and_(cls.user_id == user.id, cls.status == 'completed')).first()